lib/reply.js

```diff
@@ -387,6 +390,11 @@ function onResponseCallback (err, request, reply) {
 }
 
 function buildReply (R) {
+  if (!(new R() instanceof Reply)) {
+    console.log(R)
+    throw new Error('kaboom')
+  }
```

Comment:
I'm guessing this was just a guard for during development and should be removed?

Reply:
Good spot!